hack/test-e2e.sh

```diff
@@ -39,9 +45,9 @@ oc adm policy add-scc-to-user privileged -z rsyslog -n openshift-logging
 # allows rsyslog to query k8s api for all namespace/pod info
 oc adm policy add-cluster-role-to-user cluster-reader -z rsyslog -n openshift-logging
 
-TEST_NAMESPACE=openshift-logging go test ./test/e2e/... \
+TEST_NAMESPACE=${NAMESPACE} go test ./test/e2e/... \
   -root=$(pwd) \
-  -kubeconfig=$HOME/.kube/config \
+  -kubeconfig=${KUBECONFIG} \
```

Comment:
I see (comment from above) - is it necessary to pass in `-kubeconfig`?  It will not look at the env. var. $KUBECONFIG first, then $HOME/.kube/config?

Reply:
I have no other answer then the e2e test failed unless I provided a path where it could be something other then $HOME/.kube/config.  My assumptions is maybe the CI templates setup the environment such that it writes the config to someplace other then $HOME.  Wouldn't this be valid for a node where you do not intend to support users

Reply:
ok - weird